soc/arm/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -75,6 +75,9 @@ if DMA
 config DMA_STM32
 	default y
 
+config HEAP_MEM_POOL_SIZE
+	default 1024
```

Comment:
As I understand only DMAMUX is concerned, shouldn't we add:
```
+       depends on DMAMUX_STM32
```

Reply:
ok